android/app/src/main/java/app/organicmaps/editor/EditorFragment.java

```diff
@@ -466,7 +466,7 @@ private void initViews(View view)
     mOpeningHours.setOnClickListener(this);
     final View cardMore = view.findViewById(R.id.cv__more);
     mDescription = findInput(cardMore);
-    cardMore.findViewById(R.id.about_osm).setOnClickListener(this);
```

Comment:
Is cardMore used somewhere? Can it be deleted?

Reply:
It's used in line 468 for finding the mDescription